app/components/codelist-form.hbs

```diff
@@ -0,0 +1,176 @@
+{{#let
+  (changeset
+    @codelist this.CodelistValidations
+  ) as |codelist|
+}}
+  <BreadcrumbsItem as |linkClass|>
+    {{#if @codelist.isNew}}
+      <LinkTo
+        @route="codelists-management.new"
+        class={{linkClass}}
+      >
+        {{t "codelist.crud.new"}}
+      </LinkTo>
+    {{else}}
+      <LinkTo
+        @route="codelists-management.edit"
+        @model={{@codelist.id}}
+        class={{linkClass}}
+      >
+        {{codelist.label}}
+      </LinkTo>
+    {{/if}}
+  </BreadcrumbsItem>
+
+  <AuToolbar @border="bottom" @size="large">
+    <AuToolbarGroup>
+      <AuHeading @skin="2">
+        {{#if @codelist.isNew}}
+          {{t "codelist.crud.new"}}
+        {{else}}
+          {{t "codelist.crud.edit"}}
+        {{/if}}
+      </AuHeading>
+    </AuToolbarGroup>
+  </AuToolbar>
+
+  <div class="au-c-body-container au-c-body-container--scroll">
+    <div class="au-o-box">
+      <div class="au-o-grid">
+        <div class="au-o-grid__item au-u-2-3@medium au-u-2-5@large">
+          <form
+            class="au-c-form"
+            id="edit-road-sign-concept-form"
+            novalidate
+          >
+            {{#let codelist.error.label as |isInvalid|}}
+              <p>
+                <AuLabel @error={{isInvalid}} for="label">
+                  {{t "roadSignConcept.attr.code"}}&nbsp;
+                  <AuPill>
+                    {{t "utility.required"}}
+                  </AuPill>
+                </AuLabel>
+                <AuInput
+                  @error={{isInvalid}}
+                  id="label"
+                  required="required"
+                  value={{codelist.label}}
+                  {{on
+                    "input"
+                    (fn
+                      this.setCodelistValue
+                      codelist
+                      "label"
+                    )
+                  }}
+                />
+                {{#if isInvalid}}
+                  <AuHelpText>
+                    {{t "utility.fieldRequired"}}
+                  </AuHelpText>
+                {{/if}}
+              </p>
+            {{/let}}
+            {{#let codelist.error.codeListOptions as |isInvalid|}}
+              <p>
+                <AuLabel @error={{isInvalid}} for="values">
+                  {{t "codelist.attr.values"}}&nbsp;
+                  <AuPill>
+                    {{t "utility.required"}}
```

Comment:
About validation : indeed not validated, I was thinking that we might want some flexibility about those codelists. But maybe wrong assumption ? @nvdk @Asergey91 should we validate the codelist values ?

Reply:
Sorry should have been more clear. Now the value field shows a "required" label which might give the impression that a value needs to be entered in the field. Since it's not a traditional input, but rather something to enter data in the table, that required lable might be confusing? And since no validations are done I think removing everything related to that could be removed for this field?

Let me know if my ramblings are a bit clear :smile:

Reply:
Aaah !! Yes for sure ! Understood it the wrong way at first but you're totally right